src/lib/crypto/kimchi_bindings/stubs/src/plonk_verifier_index.rs

```diff
@@ -19,6 +25,73 @@ pub struct CamlPlonkVerificationEvals<PolyComm> {
     pub chacha_comm: Option<Vec<PolyComm>>,
 }
 
+#[derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Enum)]
```

Comment:
some comment on that: we define these types mostly on the proof system side, which is sort of "unclean" but at the same time allows us to enforce that these structs are always up to date (https://github.com/o1-labs/proof-systems/blob/master/.github/workflows/rust.yml#L30). Perhaps it'd be nice to move these there as well not to create more fragmentation